cmd/lncli/walletrpc_active.go

```diff
@@ -485,99 +483,39 @@ func bumpForceCloseFee(ctx *cli.Context) error {
 
 	// Validate the channel point.
 	channelPoint := ctx.Args().Get(0)
-	_, err := NewProtoOutPoint(channelPoint)
+	rpcChannelPoint, err := NewProtoOutPoint(channelPoint)
 	if err != nil {
 		return err
 	}
 
-	// Fetch all waiting close channels.
-	client, cleanUp := getClient(ctx)
-	defer cleanUp()
-
-	// Fetch waiting close channel commitments.
-	commitments, err := getWaitingCloseCommitments(
-		ctxc, client, channelPoint,
-	)
-	if err != nil {
-		return err
+	// Parse immediate flag (force flag was deprecated).
+	if ctx.IsSet("immediate") && ctx.IsSet("force") {
+		return fmt.Errorf("cannot set immediate and force flag at " +
+			"the same time")
 	}
+	immediate := ctx.Bool("immediate") || ctx.Bool("force")
 
 	// Retrieve pending sweeps.
 	walletClient, cleanUp := getWalletClient(ctx)
 	defer cleanUp()
 
-	sweeps, err := walletClient.PendingSweeps(
-		ctxc, &walletrpc.PendingSweepsRequest{},
-	)
+	resp, err := walletClient.BumpForceCloseFee(
+		ctxc, &walletrpc.BumpFeeRequest{
+			Outpoint:    rpcChannelPoint,
```

Comment:
as mentioned before, think we need a `BumpForceCloseFeeRequest` to make it clear that this outpoint is not the output being swept but the channel point.

Reply:
agree